hack/verify-govet.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
+  echo "Unknown go version '${GO_VERSION}', skipping go vet."
+  exit 0
+fi
+
+MODE=${1:-quiet}
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename './output' \
+        -o -wholename './_output' \
+        -o -wholename './.git' \
+        -o -wholename './release' \
+        -o -wholename './pkg/assets/bindata.go' \
+        -o -wholename './pkg/assets/*/bindata.go' \
+        -o -wholename './target' \
+        -o -wholename '*/third_party/*' \
+        -o -wholename '*/Godeps/*' \
+      \) -prune \
+    \) -name '*.go'
+}
```

Comment:
This will be 3rd place with `find_files`, if I'm counting correctly (previous are `verify-gofmt.sh`, `verify-golint.sh`, although there's one slight difference between the two `'./pkg/assets/*/bindata.go'`, but I guess that's just a leftover). I'm quite positive we should make just one function for all of `verify-*` scripts.